cfht2caom2/cfht_builder.py

```diff
@@ -107,29 +99,25 @@ def build(self, entry):
 
         # retrieve the header information, extract the instrument name
         self._logger.debug(f'Build a StorageName instance for {entry}.')
-        if (mc.TaskType.INGEST_OBS in self._config.task_types and
-                ('.fits' not in entry and not mc.StorageName.is_hdf5(entry))):
-            obs = mc.repo_get(self._repo_client, self._config.collection,
-                              entry, self._metrics)
-            instrument = md.Inst(obs.instrument.name)
-            result = cn.CFHTName(
-                obs_id=entry, instrument=instrument, entry=entry)
+        if mc.StorageName.is_hdf5(entry):
+            headers = []
```

Comment:
Replaced the `headers` declaration with an instrument assignment.

The `len(headers)` checks in the `get_instrument` method are there because of the WIRCam 'y' files, not the SITELLE hdf5 files.